lib/network/http/server.js

```diff
@@ -203,8 +204,8 @@ class Server {
     /**
      * Function called when request received
      *
-     * @param {http.IncomingMessage|https.IncomingMessage} req - Request object
-     * @param {http.ServerResponse|https.ServerResponse} res - Response object
+     * @param {http.IncomingMessage} req - Request object
+     * @param {http.ServerResponse} res - Response object
```

Comment:
Is it just standard practice to write jsdoc or we do not support receiving HTTPs requests?

Reply:
Same at other places.

Reply:
There is no https request, I looked it up, https server uses the same response and request as http.